### DISCUSSION TITLE
How to disable right-hand page TOC?

### DISCUSSION THREAD

**Discussion by retzero** [facebook/docusaurus#5088] [Q&A] [answered]
Can you please help me to remove the TOC(table of contents) sidebar?
Which configuration should I change?

**Comment by slorber** [comment, marked as answer]
https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#markdown-frontmatter

**Reply by retzero** [reply]
very thanks!

**Reply by runrohit** [reply]
this doesn't reclaim the space taken by the TOC

**Reply by slorber** [reply]
That's not true @runrohit 

Without:

![CleanShot 2025-03-13 at 17 06 04@2x](https://github.com/user-attachments/assets/d6476f4e-6efd-4fdd-b1a0-a3408cfae660)

With `hide_table_of_contents: true`

![CleanShot 2025-03-13 at 17 06 55@2x](https://github.com/user-attachments/assets/016c406a-17db-4ee6-ae1d-f34111c2f0aa)


We don't reclaim **all** the space, on purpose. I think we already reclaim too much of it and it's not even a good practice to have such large width for readability.

In any case, if you don't like what we provide, you can override it by swizzling `@theme/DocItem/Layout`
